java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/service/impl/rest/SecurityRestServiceImpl.java

```diff
@@ -39,7 +43,7 @@
   @Path("/currentuser/")
   public UserDetailsClientTo getCurrentUserDetails(@Context HttpServletRequest request) {
 
-    return TokenAuthenticationService.getUserdetailsFromToken(request.getHeader("Authorization"));
+    return this.tokenManagementService.getUserdetailsFromTokenJwt(request.getHeader("Authorization"));
```

Comment:
exactly what I guesses. IMHO does not make so much sense...
Compare this to the original implementation of this service that should still work even it never knew about JWT.